### PR TITLE
FABN-1386: Publish docs to gh-pages branch

### PR DESCRIPTION
**Pull request by bestbeforetoday**
Signed-off-by: Mark S. Lewis <mark_lewis@uk.ibm.com>

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -35,15 +35,19 @@ steps:
...
+#    NPM_TOKEN: $(NPM)
+#    PROJECT_DIR: "$(Build.Repository.LocalPath)"
+
+- script: scripts/ci_scripts/azurePublishApiDocs.sh
+  condition: and(succeeded(), eq(variables['Build.Reason'], 'IndividualCI'))
+  displayName: 'Publish API documentation'
+  env:
+    GITHUB_USER: 'Hyperledger Bot'
+    GITHUB_EMAIL: 'hlfdev.azp@gmail.com'
+    PUBLISH_URL: "https://$(GITHUB-PAT)@github.com/hyperledger/fabric-chaincode-java.git"
```
Comment (harrisob) [inline azure-pipelines.yml]: should this be fabric-chaincode-java ?

**Comment by bestbeforetoday** [comment]
Good spot, Bret!

**Pull request merged**